sdk/reservations/arm-reservations/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/arm-reservations",
   "author": "Microsoft Corporation",
   "description": "AzureReservationAPI Library with typescript type definitions for node.js and browser.",
-  "version": "5.0.0",
+  "version": "6.0.0",
```

Comment:
Are we sure this PR has breaking changes and so we need to update the major version?
At a quick glance, the changes look only additional to me

Reply:
Hi @ramya-rao-a I think the following change belongs to breaking change:
![image](https://user-images.githubusercontent.com/40835867/88253420-7d98d800-cce4-11ea-989e-cbea30644abe.png)

Reply:
From what I can see, these are used in OperationSpecs which are not exposed to the user

Reply:
You are right. I will update it.